package/MDAnalysis/core/universe.py

```diff
@@ -326,7 +326,12 @@ def __init__(self, *args, **kwargs):
             if not coordinatefile:
                 coordinatefile = None
             # parse transformations
-            self._transforms = kwargs.pop('transformations', [])
+            self._transforms = []
+            trans_arg = kwargs.pop('transformations', None)
+            if isinstance(trans_arg, list):
```

Comment:
This is going to break if the user passes a tuple, a numpy array, a generator, or anything that is not a list. I would switch the test:

```
if callable(trans_arg):
    self._transforms.append(trans_args)
elif trans_args:
    self._transforms = trans_arg
```